assets/sources/base_client.py

```diff
@@ -54,71 +54,24 @@ def wrapper(*args, **kwargs):
 
     return wrapper
 
-# Selenium needed to be cached because
-# the web scraping clients only needed one
-# selenium instance to share among the
-# subclasses, instead of multiple instances
-# one for each subclass. Having multiple
-# instances caused too many chrome instances
-# running on flip.
-@cache
-def get_selenium_webdriver(cache_id=None):
-    chrome_options = Options()
-    chrome_options.add_argument("--headless")
-    return webdriver.Chrome(chrome_options=chrome_options)
-
-
-@cache
-def killed(cache_id=None):
-    return True
-
-@cache
-def run_finally_block(cache_id=None):
-    return True
```

Comment:
Not sure I follow why all of this is taken out...We still need to cache the instance of selenium to avoid several instances of chrome being opened